idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -286,40 +298,15 @@ def build(self):
                 balance_type=MomentumBalanceType.pressureTotal,
                 has_pressure_change=self.config.vapor_side.has_pressure_change)
 
-            # TO DO : remove this warning when there is support for deltaP
-            warnings.warn("""{} WARNING! WARNING!! WARNING!!!
-                  control volume class has not implemented a method
-                  for pressure drop. Constraint for deltaP must be provided if
-                  has_pressure_change is set to True""".format(self.name))
         # consistency check
         if (self.config.vapor_side.has_pressure_change and
```

Comment:
As mentioned above, we need to clean up the config arguments so that we don't have these possible clashes.

Normally we wouldn't have a config argument for fixing pressure drop anyway - that is an standard design variable the user needs to fix themselves or add a performance equation for.